src/interpreter.rs

```diff
@@ -254,7 +254,11 @@ impl<'a, 'tcx: 'a, 'arena> Interpreter<'a, 'tcx, 'arena> {
             If { ref cond, targets: (then_target, else_target) } => {
                 let cond_ptr = try!(self.eval_operand(cond));
                 let cond_val = try!(self.memory.read_bool(cond_ptr));
-                TerminatorTarget::Block(if cond_val { then_target } else { else_target })
+                TerminatorTarget::Block(if cond_val {
+                    then_target
+                } else {
+                    else_target
+                })
```

Comment:
Seems to make this unnecessarily large. I think of the single-line `if` here like a `?:` in C.